### PR TITLE
Exclude kube-system via namespace selector from istio mutating webhook

### PR DESCRIPTION
**Pull request by piotrmsc**
**Description**

Changes proposed in this pull request:

- Add `kube-system` to namespace selector in MW istio-sidecar-injector
- Adjust kantive-eventing webhooks to allow hibernation on Gardener using evaluation profile
- Add evaluation profile for apiserver-proxy disabling hpa

**Related issue(s)**

### CODE REVIEW

**Review by Demonsthere — APPROVED** [review]
/meow

**Review by marcobebway — APPROVED** [review]

File: resources/knative-eventing/values.yaml [hunk 1]
```diff
@@ -14,6 +16,7 @@ knative-eventing:
       apiVersion: messaging.knative.dev/v1alpha1
       kind: NatssChannel
   webhook:
+    failuePolicy: Fail
```
Comment (PK85) [inline resources/knative-eventing/values.yaml:19]: failuePolicy: Fail ? :D

Comment (piotrmsc) [reply resources/knative-eventing/values.yaml:19]: :D https://github.com/kyma-project/kyma/pull/10190/files

**Pull request merged**